dbcs/honda_crv_touring_2016_can.dbc

```diff
@@ -0,0 +1,312 @@
+VERSION ""
```

Comment:
https://wenku.baidu.com/view/41c36d25ed630b1c59eeb534.html

I believe it used to be an official/internal document that leaked out, but it's dated from 2007. This document has one an error thought regarding the endianness encoding, the rest seem ok.